bin/truffleruby

```diff
@@ -16,18 +16,6 @@ if [ -n "$RUBY_BIN" ]; then
     exec "$RUBY_BIN" "$@"
 fi
 
-function mtime {
-    uname_str=$(uname)
-    if [ "$uname_str" = 'Linux' ] || [ "$uname_str" = 'SunOS' ]; then
-        stat -c '%Y' "$1" 2>/dev/null || echo 0
-    elif [ "$uname_str" = 'Darwin' ]; then
-        stat -f '%m' "$1" 2>/dev/null || echo 0
-    else
-        echo "unknown platform $uname_str" 1>&2
-        exit 1
-    fi
-}
```

Comment:
The script has 1 function and it is only used in the block so I've moved it next to its call-sites. If the launcher grows in complexity (which I hope will not) then yeah it would be good to group the functions at the top. Keeping it here for now.